test/metrics.test.js

```diff
@@ -153,7 +153,7 @@ describe('cgroup Metrics', function() {
         console.log(`CPU user count: ${metrics_object.cpuacct.stat.user}`);
         console.log(`CPU system count: ${metrics_object.cpuacct.stat.system}`);
         assert.equal(metrics_object.memory.containerUsage, 6666);
-        assert.equal(metrics_object.memory.containerUsagePercentage, 6666/9999 * 100);
+        assert.equal(metrics_object.memory.containerUsagePercentage, 6666 / 9999 * 100);
```

Comment:
nice to have:
```suggestion
        assert.equal(metrics_object.memory.containerMemoryLimit, 9999);
```

Reply:
are they all automatically included? Because I put tests in the memory.test.js file instead.